web/src/components/TraceNode/TraceNode.styled.ts

```diff
@@ -66,7 +66,7 @@ export const TraceNotch = styled.div<{spanType: SemanticGroupNames}>`
   align-items: center;
 
   span {
-    color: ${({spanType}) => getTextColor(spanType)};
+    color: ${({$spanType}) => getTextColor($spanType)};
```

Comment:
Thanks for all of these changes, this way we can get rid of the console warnings!